src/persistent.py

```diff
@@ -0,0 +1,36 @@
+import contextlib
+import os
+from typing import Optional
+
+
+class PersistentString:
+    """A string wrapper which is persistent across reboots.
+
+    Methods:
+        get(value):
+            Gets the value of the string
+        set(value):
+            Sets the value of the string and writes to disk
+    """
+
+    def __init__(self, filepath: str):
```

Comment:
```suggestion
    def __init__(self, filepath: str) -> None:
```

To be consistent with the rest of the codebase.